dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/fat/src/com/ibm/ws/jaxws/fat/HttpConduitPropertiesTest.java

```diff
@@ -429,9 +412,10 @@ private String getServletResponseMessage(String servletUrl) throws Exception {
         return result;
     }
 
-    // Waits for config change to be processed after moving the test's ibm-ws-bnd.xml file to the dropins app
-    private void waitForAppRestartAfterConfigChange() throws Exception {
-        // check logs for app restart and servlet readiness
-        server.waitForMultipleStringsInLog(++CONFIG_REFRESH_COUNT, "CWWKG0017I");
+    // Replace server configuration and wait for update to finish
+    private void swapServerConfig(String serverXMLName) throws Exception {
```

Comment:
I think the original name stucture of the method is clearer to what it does, it just needs to be updated to reflect what this method does now. I suggest using this naming for the method
```suggestion
    // Replace server configuration and wait for configuration update
    private void waitForServerConfigChange(String serverXMLName) throws Exception {
```

Reply:
We are not simply waiting for server config change, we are replacing server.xml in the same method as well.